src/frameworks/next/index.ts

```diff
@@ -136,26 +128,55 @@ export async function build(dir: string): Promise<BuildResult> {
   }
 
   const manifestBuffer = await readFile(join(dir, distDir, "routes-manifest.json"));
-  const manifest: Manifest = JSON.parse(manifestBuffer.toString());
+  const manifest = JSON.parse(manifestBuffer.toString()) as Manifest;
```

Comment:
I directed incorrectly, `manifest: Manifest` is still preferred over `as Manifest` at Google

Reply:
@jamesdaniels I replaced it with the typed `readJSON`